### PR TITLE
Packages update

### PR DESCRIPTION
**Pull request by nikmel2803**
image

### CODE REVIEW

File: src/resolvers/workspace.js [hunk 1]
```diff
@@ -207,7 +207,7 @@ module.exports = {
      * @returns {Promise<Boolean>}
      */
     async updateWorkspace(_obj, { workspaceId, name, description, image }, { user, factories }) {
-      // @makeAnIssue Create directives for arguments validation
+      // @makeAnIssue Create directives_old for arguments validation
```
Comment (ilyamore88) [inline src/resolvers/workspace.js:210]:
```suggestion
      // @makeAnIssue Create directives for arguments validation
```

Comment (ilyamore88) [reply src/resolvers/workspace.js:210]: Anyway, do we still need this comment?

File: src/resolvers/user.ts [hunk 2]
```diff
@@ -10,6 +10,7 @@ import isE2E from '../utils/isE2E';
 import { dateFromObjectId } from '../utils/dates';
 import { UserDBScheme } from '@hawk.so/types';
 import * as telegram from '../utils/telegram';
+import {MongoError} from "mongodb";
```
Comment (ilyamore88) [inline src/resolvers/user.ts:13]: Did you run ESLint?

File: src/directives/uploadImageDirective.ts [hunk 3]
```diff
@@ -1,33 +1,40 @@
...
-   * @param argument - information about argument where is image to upload
-   * @param details - details about field to resolve
-   */
-  public visitArgumentDefinition(argument: GraphQLArgument, details: {
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    field: GraphQLField<any, any>;
-    objectType: GraphQLObjectType | GraphQLInterfaceType;
-  }): GraphQLArgument | void | null {
-    const { resolve = defaultFieldResolver } = details.field;
+export default function uploadImageDirective(directiveName = 'uploadImage') {
```
Comment (ilyamore88) [inline src/directives/uploadImageDirective.ts:6]: I didn't see this spaghetti

**Review by ilyamore88 — CHANGES_REQUESTED** [review]

File: src/resolvers/user.ts (hunk 2 above)
Comment (nikmel2803) [reply src/resolvers/user.ts:13]: I will fix all eslint issues in separate request

**Review by ilyamore88 — APPROVED** [review]

**Review by GeekaN2 — APPROVED** [review]

**Pull request merged**